### ISSUE TITLE
TODO: SC2154

### ISSUE THREAD

**Issue by anishmittal2020** [koalaman/shellcheck#1345]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2148, SC2154
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
cd "${Home}" || true
```

#### Here's what shellcheck currently says:
```
Line 1	SC2148: Tips depend on target shell and yours is unknown. Add a shebang.
Line 1	SC2154: Home is referenced but not assigned.
```
#### Here's what I wanted or expected to see:

TODO: SC2154 should not be shown, as `Home` is standard variable.

**Comment by austin987** [comment]
That's expected, $Home is not a standard variable. $HOME is:
```
austin@austin2:~$ echo $Home

austin@austin2:~$ echo $HOME
/home/austin
```
